src/user/service/list.rs

```diff
@@ -0,0 +1,12 @@
+use crate::database::PooledConnection;
+use crate::errors::ServiceResult;
+use crate::graphql::model::Context;
+use crate::user::model::User;
+use diesel::prelude::*;
+
+pub(crate) fn find_all_users(context: &Context, limit: usize) -> ServiceResult<Vec<User>> {
```

Comment:
As stated above, I think that this should be put in a "service" function like this one, in order to be easily re-usable.